subcommittees/mentoring/ITProposal.md

```diff
@@ -0,0 +1,114 @@
+# Suggestions to Instructor Training
+
+**This is a rough draft.**
+
```

Comment:
This needs a tiny bit of background at the top, after skimming it, I am not completely sure which problem this should be fixing.